.travis.yml

```diff
@@ -1,25 +1,47 @@
 language: php
 
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
   - 7.0
   - hhvm
- 
+
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2"
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=1.3.*
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=2.0.*@dev
+  allow_failures:
+    - php: hhvm
+    - env: SYMFONY_VERSION=2.8.*@dev
+    - env: SYMFONY_VERSION="3.0.x-dev as 2"
 
-env:
-  - SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='~1.2'
-  - SYMFONY_VERSION='2.4.*' FOSUSERBUNDLE_VERSION='~1.2'
-  - SYMFONY_VERSION='2.3.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
-  - SYMFONY_VERSION='2.4.*' FOSUSERBUNDLE_VERSION='2.0.*@dev'
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
sure it works. But it invalidates the Travis cache (forcing to store it again) on each build, because ``$HOME/.composer/cache/repo`` contains files downloaded from Packagist and your builds are running less often than the Packagist updates (and the Symfony packages are not likely to end up in the bunch of not updated packages given the frequency of updates in the repo).
What benefits from caching on Travis are the downloaded dists